testsuite/features/support/commonlib.rb

```diff
@@ -328,3 +328,100 @@ def add_context(key, value)
   $context[$feature_scope] = {} unless $context.key?($feature_scope)
   $context[$feature_scope].merge!({ key => value })
 end
+
+# This function gets the system name, as displayed in systems list
+# * for the usual clients, it is the full hostname, e.g. suma-41-min-sle15.tf.local
+# * for the PXE booted clients, it is derived from the branch name, the hardware type,
+#   and a fingerprint, e.g. example.Intel-Genuine-None-d6df84cca6f478cdafe824e35bbb6e3b
+def get_system_name(host)
+  case host
+    # The PXE boot minion and the terminals are not directly accessible on the network,
+    # therefore they are not represented by a twopence node
```

Comment:
One less space so these line up with the `when`, unless github messed up the formatting as it sometimes does